examples/ensemble/plot_bias_variance.py

```diff
@@ -177,8 +177,8 @@ def generate(n_samples, noise, n_repeat=1):
 
     plt.subplot(2, n_estimators, n_estimators + n + 1)
     plt.plot(X_test, y_error, "r", label="$error(x)$")
-    plt.plot(X_test, y_bias, "b", label="$bias^2(x)$"),
-    plt.plot(X_test, y_var, "g", label="$variance(x)$"),
+    (plt.plot(X_test, y_bias, "b", label="$bias^2(x)$"),)
+    (plt.plot(X_test, y_var, "g", label="$variance(x)$"),)
```

Comment:
O_o

Reply:
Strange as it may seem, ruff format did the right thing here:
```python
>>> 1,
(1,)
>>> 
```

I'll remove the comma manually.